project/Win32BuildSetup/BuildSetup.bat

```diff
@@ -50,21 +51,33 @@ SET buildbinaryaddons=true
 SET exitcode=0
 SET useshell=rxvt
 SET BRANCH=na
+SET INSTALLER_ONLY=false
+SET EXE_AND_INSTALLER=false
 FOR %%b in (%1, %2, %3, %4, %5, %6) DO (
   IF %%b==clean SET buildmode=clean
   IF %%b==noclean SET buildmode=noclean
   IF %%b==noprompt SET promptlevel=noprompt
   IF %%b==nomingwlibs SET buildmingwlibs=false
   IF %%b==nobinaryaddons SET buildbinaryaddons=false
   IF %%b==sh SET useshell=sh
+  IF %%b==installeronly SET INSTALLER_ONLY=true
+  IF %%b==buildexeandinstaller SET EXE_AND_INSTALLER=true
 )
-
 SET buildconfig=Release
 set WORKSPACE=%CD%\..\..\kodi-build
 
 
   :: sets the BRANCH env var
   call getbranch.bat
+  IF %INSTALLER_ONLY%==true (
+	ECHO "Running Installer Build Only"
+	GOTO NSIS_EXE
+  )
+  
+  IF %EXE_AND_INSTALLER%==true (
```

Comment:
how is this different from the build option 'nomingwlibs'?

Reply:
Its been a while, but I'm pretty sure this is faster than `nomingwlibs`.